### PR TITLE
Enable mobile to request a specific size for keep images

### PR DESCRIPTION
**Pull request by leogrim**
@andrewconner @stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/conf/shoebox.routes [hunk 1]
```diff
@@ -238,12 +238,12 @@ GET     /m/1/libraries/:id                      @com.keepit.controllers.mobile.M
...
+GET     /m/1/libraries/:id/keeps                @com.keepit.controllers.mobile.MobileLibraryController.getKeeps(id: PublicId[Library], offset: Int ?= 0, limit: Int ?= 10, imageWidth: Option[Int] ?= None, imageHeight: Option[Int] ?= None)
 POST    /m/1/libraries/:id/invite               @com.keepit.controllers.mobile.MobileLibraryController.inviteUsersToLibrary(id: PublicId[Library])
 GET     /m/1/libraries/:id/members              @com.keepit.controllers.mobile.MobileLibraryController.getLibraryMembers(id: PublicId[Library], offset: Int ?= 0, limit: Int ?= 20)
 GET     /m/1/libraries/:id/members/suggest      @com.keepit.controllers.mobile.MobileLibraryController.suggestMembers(id: PublicId[Library], q: Option[String] ?= None, n: Int ?= 5)
 
-GET     /m/1/keeps/:id                      @com.keepit.controllers.mobile.MobileKeepsController.getKeepInfo(id: ExternalId[Keep], withFullInfo: Boolean ?= false)
+GET     /m/1/keeps/:id                      @com.keepit.controllers.mobile.MobileKeepsController.getKeepInfo(id: ExternalId[Keep], withFullInfo: Boolean ?= false, imageWidth: Option[Int] ?= None, imageHeight: Option[Int] ?= None)
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:246]: @jaredpetker @thassss 
These two routes have been updated to accept ideal image size parameters (best effort).

Comment (jaredpetker) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:246]: Awesome!, one of the only ones left that would be of good use would be the
notifications endpoint too if possible.

Comment (leogrim) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:246]: That requires much more work, @andrewconner is on it.

Comment (jaredpetker) [inline kifi-backend/modules/shoebox/conf/shoebox.routes:246]: Cool

**Pull request merged**